src/index.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>UniQ</title>
+  <meta name="viewport" content="initial-scale=1, width=device-width">
+  <link rel="stylesheet" href="styles/demo.css" />
+</head>
+<body>
+  <header id="header" class="header">
+    <div class="header__wrapper">
+      <div class="header__wrapper__logo"></div>
+      <div class="header__wrapper__breadcrumb">
+        <div class="header__wrapper__breadcrumb__step">First step:</div>
+        <div class="header__wrapper__breadcrumb__description">Describe your property</div>
+      </div>
+      <div id="toggle-menu" class="header__wrapper__mobile-buttons">
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/></svg>
+      </div>
+      <div class="header__wrapper__buttons">
+        <div class="dropdown">
+          <button class="header__wrapper__buttons__button dropdown-button">English</button>
+          <ul class="dropdown__wrapper">
+            <li class="dropdown__item">English</li>
+            <li class="dropdown__item">Español</li>
+            <li class="dropdown__item">Deutsch</li>
+            <li class="dropdown__item">Italiano</li>
+            <li class="dropdown__item">Português</li>
+          </ul>
+        </div>
+        <button class="header__wrapper__buttons__button">Help</button>
+        <div class="dropdown">
+          <button class="header__wrapper__buttons__button dropdown-button dropdown-button--active">Filipe</button>
+          <ul class="dropdown__wrapper">
+            <li class="dropdown__item">Logout</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </header>
+  <div class="progress-bar">
+    <div class="progress-bar__bar" style="width: 14%"></div>
+  </div>
+  
+  <section id="main" class="main">
+    <div class="main__wrapper">
+      <div class="main__content">
+        <div class="title">How many bedrooms in the property?</div>
+        <div class="input-container ">
+          <input type="text" class="input input--range" placeholder="2 bedrooms">
+          <div class="input-button input-button--disabled">-</div>
+          <div class="input-button">+</div>
+        </div>
+        <div class="title">How many bathrooms in the property?</div>
+        <div class="input-container">
+          <input type="text" class="input input--range" placeholder="1 bedroom">
+          <div class="input-button input-button--disabled">-</div>
```

Comment:
Please disregard structure implemented in the demo as it acts only as a placeholder.